cache/remotecache/v1/utils.go

```diff
@@ -281,7 +282,9 @@ func marshalRemote(ctx context.Context, r *solver.Remote, state *marshalState) s
 	if r.Provider != nil {
 		for _, d := range r.Descriptors {
 			if _, err := r.Provider.Info(ctx, d.Digest); err != nil {
-				return ""
+				if !cerrdefs.IsNotImplemented(err) {
```

Comment:
For others; I was curious if this should be using buildkit's or moby's `errdefs` definitions, but the `Provider.Info` interface looks to be defined by containerd; https://github.com/moby/buildkit/blob/bff1d816dd0be829bca52020092c121c6b0c9d44/vendor/github.com/containerd/containerd/content/content.go#L117-L123


(wondering if GoDoc for the interface should also mention `NotImplemented`, but maybe it's not strictly how containerd expected that to be handled other than "don't implement the interface")